stellar-contract-env-host/src/host.rs

```diff
@@ -870,6 +927,28 @@ impl Host {
             _ => Err(HostError::General("not account")),
         })
     }
+
+    #[cfg(feature = "testutils")]
+    pub fn register_test_contract(
```

Comment:
```suggestion
    pub fn register_contract(
```

Reply:
I don't want to make this change. This machinery is explicitly for test contracts. Native contracts cannot work this way.

Reply:
👍🏻 Let's circle back on how we can group and present test only functions in the SDK API.